src/solver/optim-model-filler/TimeDependentLinearExpression.cpp

```diff
@@ -210,6 +210,28 @@ TimeDependentLinearExpression TimeDependentLinearExpression::operator/(
     return out;
 }
 
+TimeDependentLinearExpression TimeDependentLinearExpression::max(
+  const TimeDependentLinearExpression& other) const
+{
+    TimeDependentLinearExpression out(*this);
+    for (std::size_t t = 0; t < size(); t++)
+    {
+        out[t] = out[t].max(other[t]);
+    }
+    return out;
+}
+
+TimeDependentLinearExpression TimeDependentLinearExpression::min(
+  const TimeDependentLinearExpression& other) const
+{
+    TimeDependentLinearExpression out(*this);
+    for (std::size_t t = 0; t < size(); t++)
+    {
+        out[t] = out[t].min(other[t]);
```

Comment:
It seems like this has useless copies (`operator=`)

```cpp
// (out[t] < other[t]) ? NOOP() : out[t] = other[t]
in_place_min(out[t], other[t]);
```
?

Reply:
useless and and inefficient:
see 
```
const LinearExpression& TimeDependentLinearExpression::operator[](std::size_t idx) const
{
    if (isConstant())
    {
        return v_[0];
    }
    else
    {
        return v_[idx];
    }
}
```

Reply:
you are right,